lenses_python/Policy.py

```diff
@@ -0,0 +1,33 @@
+from requests import *
+
+class Policy:
+    def __init__(self, token, url):
+        """
+
+        :param token:
+        """
+        self.token = token
+        self.url = url
+        self.url_extend = "/api/protection/policy"
+        self.default_headers = {'Content-Type': 'application/json', 'Accept': 'application/json',
+                                'x-kafka-lenses-token': self.token}
+
+    def view_policy(self):
+        """
+        View all versions for a policy
+
+        :return:
+        """
+        url = self.url+self.url_extend
+        response = get(url, headers=self.default_headers)
+        if response.status_code != 200:
+            raise Exception("Http status code {}.{}".format(response.status_code, response.text))
```

Comment:
we have discussed it again this is wrong by software engineering practices. as we don't have specific exceptions related to policy at least let's use `requests.HTTPError`.

Event better you can do instead of this check `response. raise_for_status()` which will automatically raise an exception. Give a try

Reply:
This will be implemented on 2.2 with a new branch